autoscheduler/frontend/src/components/LandingPage/SelectTerm.tsx

```diff
@@ -72,8 +74,8 @@ const SelectTerm: React.SFC = function App() {
           {options.map((option) => (
             <MenuItem
               key={option}
-              selected={option === 'Pyxis'}
-              onClick={handleClose}
+              selected={option === selectedTerm}
+              onClick={(): void => handleClose(option)}
```

Comment:
I had to make this into an arrow function so that I could pass in the option as an argument. If you try `onClick={handleClose(option)}`, you'll get an error because the compiler will call `handleClose(option)`, get the return value `void` and then try to set `onClick={void}`